src/env.py

```diff
@@ -630,7 +630,15 @@ def add_ipa_user_(user: dict, ipa_hostname: str = None):
                         o_userpassword=passwd)
     except pipa.exceptions.DuplicateEntry:
         env_logger.warning(f"User {username} already exists in the IPA server "
-                           f"{ipa_hostname}. Password is not changed.")
+                           f"{ipa_hostname}.")
+
+        username = f'{username}-{randint(1,1000)}'
+        env_logger.warning(f"User {username} would be added instead to "
+                           f"{ipa_hostname}.")
+        client.user_add(username, username, username, username,
```

Comment:
'client' introduced on line 626 is not descriptive and client.user_add makes feeling that we are adding user to client machine, while, ?we are using CLI to update IPA server? I would suggest something like 'server_cli' instead of 'client'

Reply:
I think the name is correct and describes what it contains - IPA meta client,  because:
1. ClientMeta does not return shell, or in other words, do not return anyt CLI. For adding IPA  we don't use ant CLI, only python IPA client API (see the documentation on [here](https://python-freeipa.readthedocs.io/en/latest/))
2. manually in the terminal on the client host you can do `ipa user-add`. the method `ClientMeta.user_add()` has the same semantic. `client` variable represents client host, from which you can add a user to the server.
Don't see any reason to rename this.